auth-source-pass.el

```diff
@@ -47,6 +47,10 @@
   "Path to the password-store folder."
   :type 'directory)
 
+(defcustom auth-source-pass-port-separator ":"
+  "Separator string between host and port in password filename."
```

Comment:
I think "entry" is more appropriate than `password` here.

Reply:
Thanks for review!
I updated the docstring.